shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Generator/Core.hs

```diff
@@ -4,6 +4,8 @@
 {-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 
+{-# OPTIONS_GHC -fno-warn-redundant-constraints #-}
```

Comment:
:thinking: I have found that GHC correctly warns about redundant `HasCallStack` constraints, so this means you have added `HasCallStack` to some functions that don't need it?